src/ShapeCrawler/Drawing/CroppingFrame.cs

```diff
@@ -11,11 +10,11 @@ namespace ShapeCrawler.Drawing;
 ///     on the given edge.
 ///     An image which is fully visible will have 0 for all values.
 /// </remarks>
-/// <param name="Left">Percentage of image along left edge of source picture which will not be displayed.</param>
-/// <param name="Right">Percentage of image along right edge of source picture which will not be displayed.</param>
-/// <param name="Top">Percentage of image from top edge of source picture which will not be displayed.</param>
-/// <param name="Bottom">Percentage of image from bottom edge of source picture which will not be displayed.</param>
-public readonly record struct CroppingFrame(decimal Left, decimal Right, decimal Top, decimal Bottom)
```

Comment:
@ashahabov I think you were more right last time, moving these to Title Case.

The compiler is generating properties for these parameters, so we'd expect them to be title case. See [Records (C# reference)](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record).

```c#
public record Person(string FirstName, string LastName);
```

Reply:
I changed my mind about this as I found that it can be managed via `.editorconfig`, without the need for a manual review.

But I agree, it doesn't look good. After all, it's not a method parameter, it's a property. So, it was reverted by https://github.com/ShapeCrawler/ShapeCrawler/commit/291c45c7decdf38089db3d47caf79ee38f1ad66b.